libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/GutenbergEditorFragment.java

```diff
@@ -339,6 +358,17 @@ public void run() {
         return view;
     }
 
+    private ArrayList<MediaOption> initOtherMediaImageOptions() {
+        ArrayList<MediaOption> otherMediaOptions = new ArrayList<>();
+
+        String packageName = getActivity().getApplication().getPackageName();
+        int stockMediaResourceId = getResources().getIdentifier("photo_picker_stock_media", "string", packageName);
```

Comment:
Something I noticed while product reviewing the PR is that this string can get pretty long in other languages (tested in french) and in that case will be cropped in the selection menu. Not a blocker but we'll probably want to address that in gutenberg (I'm thinking long pressing to make the text scroll for instance).

![image](https://user-images.githubusercontent.com/230230/67677153-978ad180-f983-11e9-8df2-c60cd6160b94.png)